resource_manager/includes/build.yml

```diff
@@ -0,0 +1,10 @@
+- name: Run the platform facts module
+  action:
```

Comment:
Any specific reason why `action` is used. I think this can be replaced with 
```
tasks: 
 - ansible.netcommon.resource_module:
      os_name: "{{ ansible_network_os }}".   # optional
      name: "{{ item }}"
      state: gathered
   register: result
```